packages/pages-components/src/components/address/methods.ts

```diff
@@ -36,6 +37,7 @@ export const getUnabbreviated = (
  * @param listings - List of available Yext Listings
```

Comment:
Add to jsdoc description that if coordinates are provided, they will be used instead of address or listings